cloud/app.py

```diff
@@ -194,7 +219,8 @@ def addInlist():
 
 
 @app.route('/getNewValues', methods=['POST'])
-def getNewValues():
+@token_required
+def getNewValues(token_id):
```

Comment:
Why is the `actuate` route not protected?

Reply:
because this is only to secure the communication between the bridge and cloud but the actuate route is one that a user will use... so yes it should also be protected but would need user management too

Reply:
ok, maybe I am missing some contextual information to judge what should be protected, sorry :sweat_smile:

Reply:
I wrote it down here #44